main.go

```diff
@@ -167,20 +182,11 @@ func main() {
 	})
 	exitOnError(err, "unable to start manager")
 
-	mcadQueueController := mcad.NewJobController(mgr.GetConfig(), cfg.MCAD, &mcadconfig.MCADConfigurationExtended{})
-	if mcadQueueController == nil {
-		// FIXME: update NewJobController so it follows Go idiomatic error handling and return an error instead of a nil object
-		os.Exit(1)
-	}
-	mcadQueueController.Run(ctx.Done())
-
-	if pointer.BoolDeref(cfg.InstaScale.Enabled, false) {
-		instaScaleController := &instascale.AppWrapperReconciler{
-			Client: mgr.GetClient(),
-			Scheme: mgr.GetScheme(),
-			Config: cfg.InstaScale.InstaScaleConfiguration,
-		}
-		exitOnError(instaScaleController.SetupWithManager(context.Background(), mgr), "Error setting up InstaScale controller")
+	certsReady := make(chan struct{})
+	if os.Getenv("ENABLE_WEBHOOKS") == "false" {
+		close(certsReady)
+	} else {
+		exitOnError(awctrl.SetupCertManagement(mgr, &cfg.AppWrapper.Config.CertManagement, certsReady), "unable to set up cert rotation")
```

Comment:
Can the setup of cert-controller be owned by the CodeFlare operator. It's going to be needed also for other controllers, so it's better `SetupCertManagement` to be in the operator itself.

Reply:
Sure.  Makes sense.

Reply:
Done.